dapr/proto/runtime/v1/dapr.proto

```diff
@@ -697,7 +697,7 @@ message StartWorkflowRequest {
   string workflow_component = 2;
   string workflow_name = 3;
   map<string, string> options = 4;
-  google.protobuf.Any input = 5;
```

Comment:
This will affect the SDK changes too, not? Why was this needed?

Reply:
Not just the Dapr client SDKs, but perhaps also the workers SDKs. `google.protobuf.Any` in the Go code creates a struct which has the property `.Value` that is a byte slice. Changing to `bytes` only, there's no struct and it's a byte slice directly.

Looks like `input` is sent as-is to components-contrib, which send it as-is to the workflow "runner".

However, I do think that `bytes` is probably better here, if we expect users to pass arbitrary JSON-encoded data.

Reply:
OK.